client/my-sites/site-settings/site-visibility/launch-site.jsx

```diff
@@ -109,15 +112,19 @@ const LaunchSite = () => {
 						</p>
 						{ isDevelopmentSite && (
 							<p>
-								{ translate(
-									'Once the site is launched, %(agencyName)s will be billed for this site in the next billing cycle.',
-									{
-										args: {
-											agencyName: agencyName,
-										},
-										comment: 'name of the agency that will be billed for the site',
-									}
-								) }
+								{ agencyLoading || agencyError
+									? translate(
+											'Once the site is launched, your agency will be billed for this site in the next billing cycle.'
+									  )
+									: translate(
+											'Once the site is launched, %(agencyName)s will be billed for this site in the next billing cycle.',
```

Comment:
It works nicely. Just wonder if the agency name could be highlighted from the rest, it gets easily lost in the middle of the text.

Reply:
That's a good point. I made the change right here since I will be merging this PR later today - so there will be time for reapproval. 🙂

Reply:
The change looks good, and my approval is still in the PR. So feel free to move forward.

Reply:
Thanks, Paulo. 🙂